src/data/map.js

```diff
@@ -1247,7 +1248,7 @@ const getSearchData = async (lat, lon, id, value, iconStyle) => {
             FROM nests
             WHERE LOWER(name) LIKE '%${sanitizedValue}%' ${pokemonSQL}
             `;
-            useManualDb = true;
+            useManualDb = dbOptions.includes('nest') ? false : true;
```

Comment:
Simplify check (remove `!` if we import manualdb useFor)
```suggestion
            useManualDb = !dbOptions.includes('nest');
```